float32 pos_estimate
float32 vel_estimate
float32 torque_target
float32 torque_estimate
float32 iq_setpoint
float32 iq_measured
ActiveErrors active_errors
AxisState axis_state
ProcedureResult procedure_result
bool trajectory_done_flag
